_data/training-modules.yaml

```diff
@@ -164,3 +164,10 @@
   videos: ''
   webpage: https://hsf-training.github.io/hsf-training-ssh-webpage/
   id: ssh
+- description: A guide through the basics of the file systems and the shell.
+  name: UNIX
+  repository: https://github.com/swcarpentry/shell-novice
+  status: alpha
```

Comment:
this should be `stable`

Reply:
You mean replacing _status: alpha_, with _status: stable_?